pdns/dnsdist.cc

```diff
@@ -1199,6 +1202,7 @@ try
   }
   
 #endif
+  ComboAddress clientAddress = ComboAddress();
```

Comment:
not really, this is because I overload the argv arguments usually used for the backend servers [here](https://github.com/PowerDNS/pdns/pull/3440/files#diff-194369649d922d5a26ab47ac9c895d33R1322)